api/src/main/java/ai/djl/ndarray/NDManager.java

```diff
@@ -161,6 +161,14 @@ static NDManager from(NDResource resource) {
      */
     ByteBuffer allocateDirect(int capacity);
 
+    /**
+     * Converts an external engine's {@link NDArray} to current engine.
```

Comment:
Does this convert or create a new one? Both description and adopt seem to imply changing.

Maybe rename to `matching(array)`.

Reply:
How about `from(NDArray)` ?

Reply:
Yeah, `from` is better. Go with that